test/pyproject.toml

```
@@ -0,0 +1,4 @@
+[project]
+name= "test_publish"
```

```suggestion
name= "test-publish-ghga-gh-action"
```

Needs to be longer otherwise name clashes occur.